packages/app-check/src/internal-api.test.ts

```diff
@@ -385,6 +387,60 @@ describe('internal api', () => {
       );
       expect(token).to.deep.equal({ token: fakeRecaptchaAppCheckToken.token });
     });
+
+    it('throttles exponentially on 503', async () => {
```

Comment:
I don't think the test tests the `exponentially` part? You should either test the exponential algorithm or update the description to accurately reflect what is being tested.

Reply:
Updated description and added a check just to make sure it's not 1 day, which would indicate it's taking the wrong path. (Because of randomization, can't check the exact time in the message, annoying to do math on the time string, provider tests do more detailed testing of the exact backoff times).